cmd/agent/ngt/sample.yaml

```diff
@@ -17,6 +17,10 @@
 ---
 version: v0.0.0
 time_zone: JST
+log:
+  mode: glg
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg?style=flat-square)
It is better to name the fields as
```yaml
logging:
  logger: glg
```

Reply:
thank you for your review!
i will fix soon